src/visualizers/P5Visualizer.ts

```diff
@@ -242,4 +242,8 @@ export abstract class P5Visualizer
        visualizer to depart a div that it's not inhabiting, or to inhabit
        a div that it's already inhabiting, nothing will happen.
     */
+
+    requestedAspectRatio(): number | undefined {
```

Comment:
Maybe add a comment to the effect that by default, P5Visualizers will use all available screen space for their visualization?